ui/app/mainui/AppMain.qml

```diff
@@ -397,6 +402,8 @@ Item {
 
     Popups {
         id: popups
+
+        sharedRootStore: RootStore
```

Comment:
This looks like we're passing some singleton here; why not make use of the `sharedRootStore` declared above?

Reply:
Thanks for your attention! Fixed (probably it was failing tests as well).